### PR TITLE
Docs getting started static diagram

### PR DESCRIPTION
**Pull request by michaellisitsa**
- [Add] - Getting started guide updated for static diagram

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,13 +2,21 @@
 
 ## Introduction
 
-Custom Diagrams is a method of plugging in visualizations into ClearCalcs calculators. They deliver a full-featured HTML, CSS and Javascript environment to develop the diagram, whilst providing tight integration with the live calculation engine of ClearCalcs.
+Custom Diagrams is a powerful method of plugging in rich visualisations into ClearCalcs calculators. They deliver a full-featured HTML, CSS and Javascript environment to develop diagrams or even other rich visualisations such as CAD viewers, graphs, images, maps and deep interactivity such as tooltips and actions, whilst taking advantage of the tight integration with the live calculation engine of ClearCalcs.
```
Comment (JayserBeam) [inline docs/README.md:5]: > develop diagrams or even other rich visualisations such as CAD viewers, graphs, images, maps and deep interactivity such as tooltips and actions, whilst taking advantage of the tight

Run on sentence and you've already used "rich visualisations".

maybe ... diagrams, other (vibrant?) visual tools/aids such as ...
I'm not sure what adjective or even if there needs to be one but you'll want to comma concat your ors and ands

File: docs/README.md [hunk 2]
```diff
@@ -2,13 +2,21 @@
...
 
-Rendering occurs in the sheet view on the client in response to new values received from the sheet.
+Try out one of our diagrams inside a real-world calculator below. Change one of the "Bolt Pattern Coordinates" and watch the diagram update.
 
-![Screenshot of the custom diagram in sheet view](_media/about/sheet-view.png)
+<div style="text-align: center;">
 
-Rendering also occurs on the ClearCalcs' servers in response to a user printing a sheet.
+[Click to Go to Interactive Example](https://clearcalcs.com/embed/steelBoltAnalysisFree/8d313ffc-6145-42a0-a86d-dddac98aa136?suppressLogo=1 ":include :type=iframe width=450px height=500px")
```
Comment (JayserBeam) [inline docs/README.md:11]: Right this replaces the entrance screenshot so you have something tangible to work with immediately

**Review by JayserBeam — CHANGES_REQUESTED** [review]
Just the wording for the introductory piece. Important enough because its the first thing people see otherwise LGTM

File: docs/README.md (hunk 2 above)
Comment (michaellisitsa) [reply docs/README.md:11]: Yes, I think it illustrates a key feature of the diagrams, that they react to the inputs in the calculator and doesn't force them to lose context to go and open one of our calculators.

File: docs/README.md (hunk 1 above)
Comment (michaellisitsa) [reply docs/README.md]: Thanks yeh that paragraph needed some work. Updated in 25e3f241e1f2444204be21da7910c8ec53b01b8c

Improved run-on sentence, put comma before the or, remove duplicate rich visualisation phrase.

**Comment by michaellisitsa** [comment]
@JayserBeam Comments addressed ready for re-review

**Review by JayserBeam — APPROVED** [review]
As stated, LGTM otherwise

**Pull request merged**